position.go

```diff
@@ -123,27 +123,27 @@ func printFile(file1 *ast.File) ([]byte, error) {
 	})
 
 	copied := 0
-	var buf2 bytes.Buffer
+	printBuf2.Reset()
```

Comment:
Can we actually reuse `printBuf1` here? It seems to me the `ast.File` returned from `parser.ParseFile` doesn't hold on to the provided source bytes, so I think `printBuf1` could be reset here and used instead of `printBuf2`?

Reply:
No can do; note how we hold onto the printBuf1 bytes via `src := printBuf1.Bytes()`.